pkg/io/video/scale_test.go

```diff
@@ -247,6 +525,103 @@ func TestScale(t *testing.T) {
 	}
 }
 
+func TestScaleFastBoxSampling(t *testing.T) {
+
+	cases := map[string]struct {
+		src           image.Image
+		width, height int
+		expected      image.Image
+	}{
+		"I420NonSquareImage": {
+			src: &image.YCbCr{
+				SubsampleRatio: image.YCbCrSubsampleRatio420,
+				Y: []uint8{
+					0xF0, 0xF0, 0x10, 0x10, 0x00, 0x00, 0x00, 0x00, 0xF0, 0xF0, 0x10, 0x10,
+					0xF0, 0xF0, 0x10, 0x10, 0x00, 0x00, 0x00, 0x00, 0xF0, 0xF0, 0x10, 0x10,
+					0x00, 0x00, 0x00, 0x00, 0x40, 0x40, 0x00, 0x00, 0x40, 0x40, 0x00, 0x00,
+					0x00, 0x00, 0x00, 0x00, 0x40, 0x40, 0x00, 0x00, 0x40, 0x40, 0x00, 0x00,
+					0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+					0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+					0x00, 0x00, 0x80, 0x80, 0x30, 0x30, 0x00, 0x00, 0x30, 0x30, 0x00, 0x00,
+					0x00, 0x00, 0x80, 0x80, 0x30, 0x30, 0x00, 0x00, 0x30, 0x30, 0x00, 0x00,
+				},
+				Cb: []uint8{
+					0x20, 0x20, 0x80, 0x80, 0x50, 0x50,
+					0x20, 0x20, 0x80, 0x80, 0x50, 0x50,
+					0x80, 0x80, 0xE0, 0xE0, 0x30, 0x30,
+					0x80, 0x80, 0xE0, 0xE0, 0x30, 0x30,
+				},
+				Cr: []uint8{
+					0xE0, 0xE0, 0x80, 0x80, 0xB0, 0xB0,
+					0xE0, 0xE0, 0x80, 0x80, 0xB0, 0xB0,
+					0xF0, 0xF0, 0x40, 0x40, 0xC0, 0xC0,
+					0xF0, 0xF0, 0x40, 0x40, 0xC0, 0xC0,
+				},
+				YStride: 12,
+				CStride: 6,
+				Rect:    image.Rect(0, 0, 12, 8),
+			},
+			width:  6,
+			height: 4,
+			expected: &image.YCbCr{
+				SubsampleRatio: image.YCbCrSubsampleRatio420,
+				Y: []uint8{
+					0xF0, 0x80, 0x08, 0x00, 0x78, 0x80,
+					0x08, 0x00, 0x20, 0x20, 0x20, 0x20,
+					0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+					0x00, 0x40, 0x58, 0x18, 0x18, 0x18,
+				},
+				Cb: []uint8{
+					0x20, 0x50, 0x68,
+					0x68, 0xB0, 0x88,
+				},
+				Cr: []uint8{
+					0xE0, 0xB0, 0x98,
+					0xD0, 0x98, 0x80,
+				},
+				YStride: 6,
+				CStride: 3,
+				Rect:    image.Rect(0, 0, 6, 4),
+			},
+		},
+	}
+
+	t.Run("FastBoxSampling", func(t *testing.T) {
```

Comment:
This Run seems redundant (same as `func TestScaleFastBoxSampling(t *testing.T)`)

Reply:
Removed in 466d14b1737a701bc31500700f8ef6f51147d76b.